Support_Your_Locals/Views/Posts/New.cshtml

```diff
@@ -1,3 +1,20 @@
-﻿<div class="container">
-    Hi. This is where you'll create a new post.
+﻿<div class="card">
+    <div class="card-header">
+        <h5>Create a New Post</h5>
+    </div>
+    <div class="card-body">
+        <div class="container">
+            <form method="post" action="New">
+                <div class="form-group">
+                    <label for="PostTitle">Title</label>
+                    <input type="text" class="form-control" id="PostTitle" placeholder="Post title">
```

Comment:
Taigi ir anąkart buvo ta pati problema: naudok asp-for ir nereikės tų kvailų id ir panašiai...